creator-node/test/prometheus.test.js

```diff
@@ -117,4 +118,16 @@ describe('test Prometheus metrics', async function () {
     assert.ok(resp.text.includes(NAMESPACE_PREFIX + '_jobs_active'))
     assert.ok(resp.text.includes(NAMESPACE_PREFIX + '_jobs_delayed'))
   })
+
+  it('Checks the duration of a bull queue job', async function () {
+    const genericBullQueue = new GenericBullQueue()
+    const job = await genericBullQueue.addTask({ timeout: 500 })
+
+    await job.finished()
+
+    const resp = await request(app).get('/prometheus_metrics').expect(200)
+    assert.ok(
+      resp.text.includes(NAMESPACE_PREFIX + '_jobs_duration_seconds_bucket')
```

Comment:
can we also check that this job gets assigned to the proper bucket? like I think it should go to the bucket above `4`

Reply:
actually nvm cus depends on nodejs event loop fuhgetaboutit